tornado/test/condition_test.py

```diff
@@ -0,0 +1,129 @@
+from datetime import timedelta
+import time
+import unittest
```

Comment:
Use `from tornado.test import unittest`; this gets you the right conditional import of unittest2 for python 2.6.